.github/CODEOWNERS

```diff
@@ -173,6 +173,7 @@
 /superwise/                              @oryan-omer support@superwise.ai @DataDog/ecosystems-review
 /syncthing/                              @sashacmc
 /taskcall/                               @taskcall support@taskcallapp.com @DataDog/ecosystems-review
+/tailscale/                              @DataDog/web-integrations
```

Comment:
Could you also add this tile to the other section to add the docs team for the manifest, metadata and README files?